lib/helpers/json-schema.js

```diff
@@ -172,6 +172,39 @@ JsonSchema.prototype.sample = function() {
 };
 
 /**
+ * Generates example data from the schema if possible. If not generate sample data.
+ *
+ * @returns {*}
+ */
+JsonSchema.prototype.example = function() {
+  if (this.schema.example)
+    return this.schema.example;
+
+  switch (this.schema.type) {
+    case 'array':
+      var item = new JsonSchema(this.schema.items);
+      return new Array(item.example());
+
+    case 'string':
+      if (this.schema.enum)
+        return this.schema.enum[Math.round(Math.random() * this.schema.enum.length)];
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'return'.](https://www.codacy.com/app/jamesmessinger/swagger-express-middleware/file/737824216/issues/source?bid=2170795&fileBranchId=2432177#l190)